test/integration/infrastructure/infrastructure_test.go

```diff
@@ -225,6 +225,19 @@ var _ = Describe("Infrastructure tests", func() {
 			Expect(err).NotTo(HaveOccurred())
 		})
 
+		It("should successfully create and delete (flow) with dualstack enabled", func() {
+			providerConfig := newProviderConfig(awsv1alpha1.VPC{
+				CIDR:             pointer.String(vpcCIDR),
+				GatewayEndpoints: []string{s3GatewayEndpoint},
```

Comment:
Why is this necessary?

```suggestion
```

Reply:
This is needed as in verifyCreation the vpc endpoint is checked https://github.com/DockToFuture/gardener-extension-provider-aws/blob/8c611e76de08b59c9818e7dce46cfd8e7fac349d/test/integration/infrastructure/infrastructure_test.go#L933
It's the same for all tests.